src/shippingservice/main.go

```diff
@@ -68,7 +78,12 @@ func main() {
 	if err != nil {
 		log.Fatalf("failed to listen: %v", err)
 	}
-	srv := grpc.NewServer(grpc.StatsHandler(&ocgrpc.ServerHandler{}))
+	// TOOD: replace ocgrpc with automatic OpenTelemetry grpc metrics collector
```

Comment:
Is this the automatic OpenTelemetry metrics collector?

Reply:
nope, it doesn't exist yet :P that's why TODO